Bottom: f6db665131989166508d7508428282697dffd14a
Top:    abb18303ae8bee4adfab6f1c69a9261dee7b6358
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 02:44:09 +0000

Refresh of configure-ac-save-and-restore--0

---

diff --git a/configure.ac b/configure.ac
index 7e8348b..652f08a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1107,7 +1107,7 @@ AC_DEFINE_UNQUOTED([COORD_MAX],[$COORD_MAX],
 
 # ------------- Complete set of CFLAGS and LIBS -------------------
 
-CFLAGS="$X_CFLAGS $DBUS_CFLAGS $GLIB_CFLAGS $GTK_CFLAGS $GD_CFLAGS $CAIRO_CFLAGS $GTKGLEXT_CFLAGS $GLU_CFLAGS $GL_CFLAGS"
+CFLAGS="$CFLAGS $X_CFLAGS $DBUS_CFLAGS $GLIB_CFLAGS $GTK_CFLAGS $GD_CFLAGS $CAIRO_CFLAGS $GTKGLEXT_CFLAGS $GLU_CFLAGS $GL_CFLAGS"
 LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_LIBS $INTLLIBS $CAIRO_LIBS $GTKGLEXT_LIBS $GLU_LIBS $GL_LIBS"
